engine/src/uci/crazyara.cpp

```diff
@@ -150,7 +151,13 @@ void CrazyAra::uci_loop(int argc, char *argv[])
 #ifdef USE_RL
         else if (token == "selfplay")   selfplay(is);
         else if (token == "arena")      arena(is);
-#endif
+    #ifdef MODE_STRATEGO
+        // Test if the new modes are also usable for chess and others
+        else if (token == "mctsmatch")   mctsarena(is);
+        else if (token == "mctstournament")   mctstournament(is, 100);
```

Comment:
Should we also allow starting a given amount of matches, instead of only 100, here?
Similiar to the original `arena` command.

Reply:
Sure